postman_tests/lib/es.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
+import os
+import glob
+import json
+import csv
+import requests
+from . import common
+
+
+def get_es_api_url(config_path):
+    if os.path.exists(f"{config_path}/dev.json"):
+        config_file_path = f"{config_path}/dev.json"
+    else:
+        config_file_path = f"{config_path}/prod.json"
+    with open(config_file_path, 'r') as f:
+        data = json.load(f)
+    base_url = data["es_host"]
```

Comment:
Incorrect naming, this is just NER server url, ES could be anywhere else, we are not interacting directly with ES.
It should be taken directly from `values` where `key` is `url`